server/index.js

```diff
@@ -130,6 +132,15 @@ app.get("/v2/queryByStoreId", async (req, res) => {
   res.send({ location: params.location, results });
 });
 
+const cbOptions = {
+  timeout: 5000, // If our function takes longer than 5 seconds, trigger a failure
+  errorThresholdPercentage: 10, // When 10% of requests fail, trip the circuit
+  resetTimeout: 60000 // After 60 seconds, try again.
+};
+
+const breaker = new CircuitBreaker((params) => listing.findStoreListings(params), cbOptions);
```

Comment:
ooh, looks fancy. can we move this to within the `v3/query` route if it's only being used there?

Reply:
It will get initialized again and again. It needs to be composed once

Reply:
Ah ok got it 👍